sparkbox/sparkpress-wordpress-starter
# Add twig templating

**Changes:**
- Replaces all PHP templates with Twig templates.
- Adds documentation on changes

**To verify**
- Install on master and verify everything exists and performs as intended
- Switch to this branch and verify nothing breaks and that twig makes sense

Reminder to check:
- Page Template
- Custom Post Type
- Archive
- Post
- Admin Section
